ci/properties/openshift.properties.json

```diff
@@ -3,5 +3,5 @@
     "description": "Build a Docker-based project and deploy it to OpenShift.",
     "creator": "Red Hat",
     "iconName": "openshift",
-    "categories": [ "Dockerfile" ]
+    "categories": ["Dockerfile","deployment" ]
```

Comment:
```suggestion
    "categories": ["Deployment", "Dockerfile"]
```

Reply:
I see that the Category(["Code Scanning"](https://github.com/actions/starter-workflows/blob/216e9acdacbbd16cfdf1ec30c3bc98bcdefb7580/code-scanning/properties/flawfinder.properties.json#L6), ["Automation"](https://github.com/actions/starter-workflows/blob/216e9acdacbbd16cfdf1ec30c3bc98bcdefb7580/automation/properties/greetings.properties.json#L5)) is added 1st in the list.
The category is also capitalized.

Reply:
Let us keep the order this way for the time being as the plan is to make only additional changes and not make any change in the existing order. 

Also, I plan to make categories case-insensitive. Will capitalize the category keyword as suggested. Should work either way.

Reply:
FYI that the first category listed is considered the "primary category" and hence keeping "Dockerfile" first in the list to maintain the status quo . [Link](https://github.com/github/github/blob/1bb860fea968de947cb4b740aab65f67404a01c9/app/view_models/actions/starter_workflows_view.rb#L156)